docs/plugins/convert.rst

```diff
@@ -54,6 +54,11 @@ instead, passing ``-H`` (``--hardlink``) creates hard links.
 Note that album art embedding is disabled for files that are linked.
 Refer to the ``link`` and ``hardlink`` options below.
 
+by default, any extra files in the destination directory are left untouched. To
+remove any files that have no matching library file after converting, use the
```

Comment:
I'm a little confused by this wording: what exactly is a "matching library file"? Maybe some examples would help.